### ISSUE TITLE
SI-7420: applyDynamic does not support passing a vararg parameter

### ISSUE THREAD

**Issue by seratch** [scalikejdbc/scalikejdbc#253]
Scala lang's unfixed issue.

<del>https://issues.scala-lang.org/browse/SI-7420 </del>
https://github.com/scala/bug/issues/7420

https://gist.github.com/seratch/28c07cbe788c467c9a48

**Comment by ocadaruma** [comment]
I encountered this problem when using `sqls.joinWithAnd`.
How about providing `joinWithAnd` overload like this ?

```scala
def joinWithAnd[A](parts: A)(implicit ev: A =:= Seq[SQLSyntax]): SQLSyntax = ...
```

**Comment by xuwei-k** [comment]
fixed since Scala 2.12.5 https://github.com/scala/scala/pull/6249
